<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PublicationWidget</class>
 <widget class="QWidget" name="PublicationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Basics</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_3">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Entry type:</string>
         </property>
         <property name="buddy">
          <cstring>editEntryType</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="KComboBox" name="editEntryType">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Title:</string>
         </property>
         <property name="buddy">
          <cstring>editTitle</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="StringEdit" name="editTitle"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Authors:</string>
         </property>
         <property name="buddy">
          <cstring>editAuthors</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="ContactEdit" name="editAuthors"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Editors:</string>
         </property>
         <property name="buddy">
          <cstring>editEditor</cstring>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="ContactEdit" name="editEditor"/>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Publication Date:</string>
         </property>
         <property name="buddy">
          <cstring>editDate</cstring>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="StringEdit" name="editDate"/>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Publisher:</string>
         </property>
         <property name="buddy">
          <cstring>editPublisher</cstring>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="ContactEdit" name="editPublisher"/>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="label_28">
         <property name="text">
          <string>Institution:</string>
         </property>
         <property name="buddy">
          <cstring>editCrossref</cstring>
         </property>
        </widget>
       </item>
       <item row="16" column="0">
        <widget class="QLabel" name="label_27">
         <property name="text">
          <string>Organization:</string>
         </property>
         <property name="buddy">
          <cstring>editCrossref</cstring>
         </property>
        </widget>
       </item>
       <item row="16" column="1">
        <widget class="StringEdit" name="editOrganization"/>
       </item>
       <item row="21" column="1">
        <widget class="FileObjectEdit" name="editFileObject"/>
       </item>
       <item row="23" column="1">
        <widget class="FileObjectEdit" name="editRemoteObject"/>
       </item>
       <item row="14" column="1">
        <widget class="StringEdit" name="editInstitution"/>
       </item>
       <item row="19" column="1">
        <widget class="StringEdit" name="editCrossref"/>
       </item>
       <item row="19" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Crossref:</string>
         </property>
         <property name="buddy">
          <cstring>editCrossref</cstring>
         </property>
        </widget>
       </item>
       <item row="21" column="0">
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>File:</string>
         </property>
         <property name="buddy">
          <cstring>editFileObject</cstring>
         </property>
        </widget>
       </item>
       <item row="23" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Remote File:</string>
         </property>
         <property name="buddy">
          <cstring>editFileObject</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Extra</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Bookseries:</string>
         </property>
         <property name="buddy">
          <cstring>editSeries</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="SeriesEdit" name="editSeries"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Edition:</string>
         </property>
         <property name="buddy">
          <cstring>editEdition</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="StringEdit" name="editEdition"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Journal:</string>
         </property>
         <property name="buddy">
          <cstring>editJournal</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="JournalEdit" name="editJournal"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Volume:</string>
         </property>
         <property name="buddy">
          <cstring>editVolume</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="VolumeNumberEdit" name="editVolume"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Number:</string>
         </property>
         <property name="buddy">
          <cstring>editNumber</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="VolumeNumberEdit" name="editNumber"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Publication Method:</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="buddy">
          <cstring>editHowPublished</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="StringEdit" name="editHowPublished"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Type:</string>
         </property>
         <property name="buddy">
          <cstring>editType</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="StringEdit" name="editType"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>Copyrigth:</string>
         </property>
         <property name="buddy">
          <cstring>editCopyright</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="StringEdit" name="editCopyright"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>Last Accessed:</string>
         </property>
         <property name="buddy">
          <cstring>editCopyright</cstring>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="StringEdit" name="editLastAccessed"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Identification</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>E-Print:</string>
         </property>
         <property name="buddy">
          <cstring>editEprint</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="StringEdit" name="editEprint"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>ISBN:</string>
         </property>
         <property name="buddy">
          <cstring>editISBN</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="StringEdit" name="editISBN"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>ISSN:</string>
         </property>
         <property name="buddy">
          <cstring>editISSN</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="StringEdit" name="editISSN"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>LCCN:</string>
         </property>
         <property name="buddy">
          <cstring>editLCCN</cstring>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="StringEdit" name="editLCCN"/>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>MRNumber:</string>
         </property>
         <property name="buddy">
          <cstring>editMRNumber</cstring>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="StringEdit" name="editMRNumber"/>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>DOI:</string>
         </property>
         <property name="buddy">
          <cstring>editDOI</cstring>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="StringEdit" name="editDOI"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Note</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Note:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="noteEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Annotation:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="annoteEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addReference">
       <property name="minimumSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>StringEdit</class>
   <extends>QLineEdit</extends>
   <header>propertywidgets/stringedit.h</header>
   <slots>
    <signal>widgetEnabled(bool)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>ContactEdit</class>
   <extends>QLineEdit</extends>
   <header>propertywidgets/contactedit.h</header>
   <slots>
    <signal>widgetEnabled(bool)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>SeriesEdit</class>
   <extends>QLineEdit</extends>
   <header>propertywidgets/seriesedit.h</header>
  </customwidget>
  <customwidget>
   <class>JournalEdit</class>
   <extends>QLineEdit</extends>
   <header>propertywidgets/journaledit.h</header>
  </customwidget>
  <customwidget>
   <class>VolumeNumberEdit</class>
   <extends>QLineEdit</extends>
   <header>propertywidgets/volumenumberedit.h</header>
  </customwidget>
  <customwidget>
   <class>FileObjectEdit</class>
   <extends>QLineEdit</extends>
   <header>propertywidgets/fileobjectedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>editTitle</sender>
   <signal>widgetEnabled(bool)</signal>
   <receiver>label_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editAuthors</sender>
   <signal>widgetEnabled(bool)</signal>
   <receiver>label_4</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editEditor</sender>
   <signal>widgetEnabled(bool)</signal>
   <receiver>label_12</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editDate</sender>
   <signal>widgetEnabled(bool)</signal>
   <receiver>label_5</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editPublisher</sender>
   <signal>widgetEnabled(bool)</signal>
   <receiver>label_10</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editCrossref</sender>
   <signal>widgetEnabled(bool)</signal>
   <receiver>label_18</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editSeries</sender>
   <signal>widgetEnabled(bool)</signal>
   <receiver>label_6</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editEdition</sender>
   <signal>widgetEnabled(bool)</signal>
   <receiver>label_11</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editJournal</sender>
   <signal>widgetEnabled(bool)</signal>
   <receiver>label_7</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editVolume</sender>
   <signal>widgetEnabled(bool)</signal>
   <receiver>label_8</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editNumber</sender>
   <signal>widgetEnabled(bool)</signal>
   <receiver>label_9</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editHowPublished</sender>
   <signal>widgetEnabled(bool)</signal>
   <receiver>label_14</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editType</sender>
   <signal>widgetEnabled(bool)</signal>
   <receiver>label_17</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editCopyright</sender>
   <signal>widgetEnabled(bool)</signal>
   <receiver>label_24</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editFileObject</sender>
   <signal>widgetEnabled(bool)</signal>
   <receiver>label_16</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editEprint</sender>
   <signal>widgetEnabled(bool)</signal>
   <receiver>label_13</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editISBN</sender>
   <signal>widgetEnabled(bool)</signal>
   <receiver>label_19</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editISSN</sender>
   <signal>widgetEnabled(bool)</signal>
   <receiver>label_20</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editLCCN</sender>
   <signal>widgetEnabled(bool)</signal>
   <receiver>label_21</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editMRNumber</sender>
   <signal>widgetEnabled(bool)</signal>
   <receiver>label_22</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editDOI</sender>
   <signal>widgetEnabled(bool)</signal>
   <receiver>label_23</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
